### PR TITLE
feat: set(not takeover) window.ethereum by default

### PR DESCRIPTION
**Pull request by yqrashawn**
PORTAL-2918

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/915)

### CODE REVIEW

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-2918 当未安装 metamask，即使 fluent 托管按钮没打开，连接 metamask 也能调起 fluent，默认不托管 metamask](https://app.clickup.com/t/3719212/PORTAL-2918)

File: packages/inpage/setup-inpage-provider.test.js [hunk 1]
```diff
@@ -15,5 +15,12 @@ describe('inpage', function () {
       expect(eventName).toBe('message')
       expect(window.conflux).toBeDefined()
     })
+
+    it('shouldn not setup the provider on window.ethereum when already defined', async function () {
```
Comment (angelia-yuqi-personal) [inline packages/inpage/setup-inpage-provider.test.js:19]: typo

**Review by angelia-yuqi-personal — CHANGES_REQUESTED** [review]

**Comment by yqrashawn** [comment]
typeo fixed

**Review by angelia-yuqi-personal — APPROVED** [review]

**Review by zctocm — APPROVED** [review]
> Reviewed 3 of 4 files at r1, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/conflux-chain/helios/915)* status: all files reviewed, 1 unresolved discussion (waiting on @yqrashawn)

**Pull request merged**